.github/workflows/library.yml

```diff
@@ -353,27 +353,29 @@ jobs:
           git config --local user.email "propagate-bot@pymedphys.com"
           git config --local user.name "PyMedPhys Propagate Bot"
 
+          git fetch origin $BRANCH
+          git checkout $BRANCH
```

Comment:
and this is how it avoids the merge in to main, the commit is now going in to $BRANCH
If that's the case, great.
If not, some explanatory text would be useful.

Reply:
> and this is how it avoids the merge in to main, the commit is now going in to $BRANCH

Yup 🙂, that's the case.